src/pages/home/HomeBlogPreview.jsx

```diff
@@ -2,26 +2,36 @@ import style from "../../style";
 import { SectionBottom, SectionTop } from "../../layout/Section";
 import { posts } from "../../posts/postTransformers";
 import useDisplayCategory from "../../hooks/useDisplayCategory";
-import Carousel from "../../layout/Carousel";
 import moment from "moment";
-import React, { useState } from "react";
+import React from "react";
 import useCountryId from "../../hooks/useCountryId";
 import EmphasisedLink from "../../layout/EmphasisedLink";
 import { FileImageOutlined } from "@ant-design/icons";
 
 export default function HomeBlogPreview() {
   const countryId = useCountryId();
   const featuredPosts =
-    posts.filter(
+    posts
+      .filter(
+        (post) =>
+          post.tags.includes("featured") &&
+          (post.tags.includes(countryId) || post.tags.includes("global")),
+      )
+      .sort((a, b) => b.date - a.date) || [];
+  const otherPosts = posts
+    .filter(
       (post) =>
-        post.tags.includes("featured") &&
-        (post.tags.includes(countryId) || post.tags.includes("global")),
-    ) || [];
-  const allPosts = posts.filter(
-    (post) =>
-      (post.tags.includes(countryId) || post.tags.includes("global")) &&
-      !featuredPosts.includes(post),
-  );
+        (post.tags.includes(countryId) || post.tags.includes("global")) &&
+        !featuredPosts.includes(post),
+    )
+    .sort((a, b) => b.date - a.date);
+
+  // Combine featured posts and other posts
+  const allCombinePosts = featuredPosts.concat(otherPosts);
+
+  // Extract the first post and the rest of the posts
+  const [FirstPost, ...allPosts] = allCombinePosts;
```

Comment:
Instead of extracting the first post here, could you remove this line, remove the references to `FirstPost`, and just have each component below pass in `allPosts[0]` as their featured post? I'll demonstrate it once below on one of the components, but it should be the same for the others.